src/diffusers/pipelines/cogvideo/pipeline_cogvideox_video2video.py

```diff
@@ -207,7 +207,9 @@ def __init__(
             tokenizer=tokenizer, text_encoder=text_encoder, vae=vae, transformer=transformer, scheduler=scheduler
         )
         self.latents_in_channels = (
-            self.transformer.config.in_channels if hasattr(self, "transformer") and self.transformer is not None else 16
+            self.transformer.config.in_channels
+            if hasattr(self, "transformer") and self.transformer is not None
+            else 16
```

Comment:
Could we make similar changes to the other T2V and I2V pipelines?

Also, this value should be 32 for the I2V pipeline instead of 16.